torch/utils/hooks.py

```diff
@@ -34,13 +45,20 @@ def __setstate__(self, state) -> None:
         self.id = state[1]
         RemovableHandle.next_id = max(RemovableHandle.next_id, self.id + 1)
 
-    def __enter__(self) -> 'RemovableHandle':
+        self.kwargs_dict_ref = (
+            None
+            if len(state) < 3
+            else weakref.ref(dict() if state[2] is None else state[2])
```

Comment:
Why dict() nad not OrderedDict() like the one above?

Reply:
I thought we don't need order for this one. Let me change that to `OrderedDict` to keep things consistent.

Reply:
Same as with the other one tbh.